docs/_docs/installation.md

```diff
@@ -10,16 +10,31 @@ encountered and how we might make the process easier.
 
 ### Requirements
 
-Installing Jekyll is easy and straight-forward, but there are a few
-requirements you’ll need to make sure your system has before you start.
+Installing Jekyll ought to be straight-forward if all requirements are met.
+Before you start, make sure your system has the following:
 
-- [Ruby](https://www.ruby-lang.org/en/downloads/) (including development
+- Linux, Unix, or macOS
+- [Ruby](https://www.ruby-lang.org/en/downloads/) (including all development
   headers, v1.9.3 or above for Jekyll 2 and v2 or above for Jekyll 3)
 - [RubyGems](https://rubygems.org/pages/download)
-- Linux, Unix, or macOS
-- [NodeJS](https://nodejs.org/), or another JavaScript runtime (Jekyll 2 and
-earlier, for CoffeeScript support).
-- [Python 2.7](https://www.python.org/downloads/) (for Jekyll 2 and earlier)
+
+#### Only required for Jekyll 2 and earlier
+- [NodeJS](https://nodejs.org/), or another JavaScript runtime (for CoffeeScript support).
+- [Python 2.7](https://www.python.org/downloads/)
+
+<div class="note info">
+  <h5>Running Jekyll on Ubuntu</h5>
+  <p>
+    Users of Jekyll on Ubuntu have reported encountering 
+    <i>Could not locate Gemfile or .bundle/ directory</i> error messages at the 
+    <code>bundle exec jekyll serve</code> step in the <a href="../quickstart/">Quick-start guide</a>.
```

Comment:
> ```text
> Fetching: ffi-1.9.14.gem (100%)
> Building native extensions.  This could take a while...
> ERROR:  Error installing jekyll:
> 	ERROR: Failed to build gem native extension.
> 
>     current directory: /var/lib/gems/2.3.0/gems/ffi-1.9.14/ext/ffi_c
> /usr/bin/ruby2.3 -r ./siteconf20170113-5768-tdfh0m.rb extconf.rb
> mkmf.rb can't find header files for ruby at /usr/lib/ruby/include/ruby.h
> 
> extconf failed, exit code 1
> 
> Gem files will remain installed in /var/lib/gems/2.3.0/gems/ffi-1.9.14 for inspection.
> Results logged to /var/lib/gems/2.3.0/extensions/x86_64-linux/2.3.0/ffi-1.9.14/gem_make.out
>```